edward/stats/distributions.py

```diff
@@ -142,16 +111,69 @@ def rvs(self, p, size=1):
         return x
 
     def logpmf(self, x, p):
+        """Logarithm of probability mass function
+
+        Parameters
+        ----------
+        x : np.array or tf.Tensor
+            If univariate distribution, can be a scalar, vector, or matrix.
+            If multivariate distribution, can be a vector or matrix.
+        p : float
```

Comment:
Following the template description, the type is actually `float, np.array, or tf.Tensor`. Same goes with all the other distribution methods with `logpmf()`/`logpdf()`. And this is true for both the observation `x` and any parameters.